galley/pkg/runtime/source_test.go

```diff
@@ -0,0 +1,146 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package runtime
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gogo/protobuf/types"
+
+	"istio.io/istio/galley/pkg/runtime/resource"
+)
+
+func TestInMemory_Start_Empty(t *testing.T) {
+	i := NewInMemorySource()
+	ch, err := i.Start()
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	actual := logChannelOutput(ch, 1)
```

Comment:
optional: got/want instead of expected/actual would be fewer characters to type.

Reply:
I am used to actual/expected nomenclature, and have been using it throughout the code. :)